packages/graphiql/cypress/e2e/tabs.cy.ts

```diff
@@ -2,8 +2,8 @@ describe('Tabs', () => {
   it('Should store editor contents when switching between tabs', () => {
     cy.visit('/?query=');
 
-    // Assert that no tab visible when there's only one session
-    cy.get('#graphiql-session-tab-0').should('not.exist');
+    // Assert that tab visible when there's only one session
+    cy.get('.graphiql-tab-button').eq(0).should('exist');
```

Comment:
was false negative after https://github.com/graphql/graphiql/pull/3713 merged, if we add `cy.wait(1000)` right before it failed